### ISSUE TITLE
Test Generators

### ISSUE THREAD

**Issue by MattCheely** [MattCheely/generator-elm-parcel#10]
It would be good to have some unit tests for the generators to make sure they're doing things as expected.

**Comment by kalutheo** [comment]
Yes we could add jest and maybe typescript to have type safety in our javascript code
